index.html

```diff
@@ -9,24 +9,71 @@
   <body>
     
      <div class="wrapper">
-      <h3>All Awesome Books</h3> 
-      <!---<div class="container-book"></div>--> 
-      <div class="book-from-storage"> </div> 
+        <header>
+          <nav class="menu-bar">
+            <li class="logo">
+              <a href="#" class="menu-item-link">
+                Awesome Books
+              </a>
+            </li>
+            <ul class="menus">
+              
+              <li class="menu-item">
+                <a href="#list" class="menu-item-link list-anchor">
+                  List
+                </a>
+              </li>
+              <li class="menu-item">
+                <a href="#add-new" class="menu-item-link add-new-anchor">
+                  Add new
+                </a>
+              </li>
+              <li class="menu-item">
+                <a href="#contact" class="menu-item-link contact-anchor">
+                  Contact
+                </a>
+              </li>           
 
-      <div class="hor-line"></div>
+            </ul>
+          </nav>
+        </header> 
+        <span class="time">March 19th 2021 9:25:32</span>
```

Comment:
- [ ] Kindly ensure that you have positioned this time at the right side as it is in the wireframe.
- [ ] Also, kindly note that date and time are required to be generated by the Luxon library. Kindly ensure that the date and time is also no longer hard coded as it is right now.

**This is your design:**

![image](https://user-images.githubusercontent.com/86318284/193426824-f9c69f4e-66d4-4c7e-89f5-e19ef5dbad76.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/193426836-9c430568-4f64-49ec-a995-a279967c21ff.png)